src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Maybe have header tags within the Header component as opposed to the App component?

Reply:
Great suggestion. Ill get on that.